\documentclass[letterpaper]{article}
\usepackage[letterpaper,top=1in,bottom=1in,left=1in,right=1in]{geometry}
\usepackage{graphicx}
%\usepackage{times}      % Loads the Times-Roman Fonts
%\usepackage{mathptmx}   % Loads the Times-Roman Math Fonts
\usepackage{helvet}
\usepackage[sf,pagestyles]{titlesec}
\usepackage{lineno}
\usepackage{setspace}

\setlength{\parindent}{0em}
\setlength{\parskip}{1em}

\begin{document}

\doublespace

\section*{Robustness of phylogenetic analysis for detecting clusters of new HIV infections}

\textbf{Authors:}

August Guang$^1$, Mark Howison$^2$, Mia Coetzer$^3$, Lauren Ledingham$^3$, Matt D'Antuono$^3$,
Philip A. Chan$^3$, Charles Lawrence$^4$, Casey W. Dunn$^5$, Rami Kantor$^3$

\small $^1$ Computing and Information Services, Brown University, Providence, RI, USA

\small $^2$ Research Improving People's Lives, Providence, RI, USA

\small $^3$ Division of Infectious Diseases, The Alpert Medical School, Brown University, Providence, RI, USA

\small $^4$ Division of Applied Mathematics, Brown University, Providence, RI, USA

\small $^5$ Department of Ecology and Evolutionary Biology, Yale University, New Haven, CT, USA

\linenumbers

\section*{Abstract}

\textbf{Background:} Phylogenetic analysis of HIV sequences obtained as part of clinical care is increasingly applied to detect clustering of new HIV infections and inform public health interventions to disrupt transmission. Conventional approaches summarize the within-host HIV diversity using only a single consensus sequence of only the HIV pol gene per individual, typically from Sanger or next-generation sequencing (NGS).

\textbf{Methods:} In all newly HIV diagnosed individuals in the first half of 2013 from the largest HIV center in Rhode Island, USA, we evaluate the robustness of the consensus approach and the potential benefit of considering full within-host HIV diversity, available via NGS, and of near whole HIV genome for phylogenetic inference. We compare Sanger and NGS-derived pol consensus sequences to an alternate approach that samples many sequences per individual from a profile hidden Markov model of their NGS data for pol and near-whole HIV genomes.

\textbf{Results:} The space of phylogenies inferred from sampling is multi-modal, suggesting that a consensus-inferred phylogeny is not an appropriate summary of within-host variation. Cluster inference differs in phylogenies from consensus sequences in three clinically-relevant regions (prrt, int, env) versus the whole genome, and using Sanger versus NGS data.

\textbf{Discussion:} The choice of gene region and sequencing and summary methods affects the detection of clusters, and should be considered carefully in public health applications of phylogenetic analysis to disrupt HIV transmission.

\section*{Background}

\section*{Methods}

\section*{Results}

\section*{Discussion}

\bibliography{references}

\end{document}
